### DISCUSSION TITLE
Is there any way to get the raw data?

### DISCUSSION THREAD

**Discussion by avriltank** [alpinejs/alpine#2935] [1. Help]
$data returns the proxy object,how can i get the raw object?By vue,we can use toRaw method

**Comment by lfavole** [comment]
You can use `Alpine.raw` but it is not documented.

**Comment by ekwoka** [comment]
As mentioned, `Alpine.raw`.

Alpine uses an older Vue Reactivity, so it behaves the same there. Alpine.raw is just a reexport of the one from vue reactivity. Additionally, due to vue reactivity and it's behavior, you can also do `myobject.__raw__` I believe it is. since that is specially handled in the reactive proxy.
